src/repository/crud/user.py

```diff
@@ -9,11 +9,8 @@ async def save(user: User):
         query = "INSERT INTO user (username, password, email, full_name, address, phone) VALUES (%s, %s, %s, %s, %s, %s)"
         user_data = (
             user.username,
-            user.password,
             user.email,
```

Comment:
Why do you remove this fields? Is only for know it . . .

Reply:
Finally I replaced the "username, password, email, full_name, address, phone" in user table with "name, email, password, address, phone", in which the address and phone field are optional